Makefile

```diff
@@ -1,24 +1,23 @@
 .PHONY: requirements
 
 requirements: requirements.js
-	pip install -q -r requirements/base.txt --exists-action w
+	pip install .
```

Comment:
I think that's ok; let's just be sure that it can be picked up by gemnasium or VersionEye. Could you check that out?